concourse/scripts/self-update-pipeline.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Required variables are:
+# - DEPLOY_ENV
+# - AWS_ACCOUNT
+# - SELF_UPDATE_PIPELINE
+#
+# Optional variables:
+# - BRANCH
+
+set -u
+set -e
+
+if [ ! -d "./paas-bootstrap" ]; then
+  echo "Resource paas-bootstrap must be checked out"
+  exit 1
+fi
+
+if [ "${TARGET_CONCOURSE}" == "bootstrap" ]; then
+  echo "Bootstrap Concourse should not self-update, as this requires access to \`aws sts get-caller-identity\`. Skipping."
+  exit 0
+fi
+
+if [ "${SELF_UPDATE_PIPELINE}" != "true" ]; then
```

Comment:
Can we reverse this conditional. As it stands, it's effectively an `unless...else...` clause, which takes a moment to parse.